qutebrowser/browser/commands.py

```diff
@@ -985,6 +987,37 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         url = objreg.get('quickmark-manager').get(name)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', name='select-follow',
+                       scope='window')
+    def select_follow(self, tab=False):
```

Comment:
What do you think about the command name? I think `follow-selected` or so would make more sense.